backend/benefit/helsinkibenefit/settings.py

```diff
@@ -107,6 +109,7 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "simple_history.middleware.HistoryRequestMiddleware",
```

Comment:
Oh, I think I forgot to add this into kesäseteli. Kudos for reading the documentation :grin:

Reply:
:-)